src/db-up/setup.py

```diff
@@ -28,6 +29,9 @@
     'psycopg2-binary==2.9.1'
 ]
 
+if sys.platform != 'darwin':
```

Comment:
Why darwin doesn't need it?

Reply:
Actually darwin also need. But the installation in darwin depends on `brew install freetds` first.